src/Symfony/Component/PropertyAccess/StringUtil.php

```diff
@@ -11,122 +11,18 @@
 
 namespace Symfony\Component\PropertyAccess;
 
+use Symfony\Component\Inflector\Inflector;
+
 /**
  * Creates singulars from plurals.
  *
  * @author Bernhard Schussek <bschussek@gmail.com>
+ *
+ * @deprecated Deprecated since version 3.1, to be removed in 4.0. Use {@see \Doctrine\Common\Inflector\Inflector} instead.
```

Comment:
Telling people to use Doctrine Inflector does not make sense here as the feature is not the same. And in fact, we are creating a new component exactly for this reason: we cannot use Doctrine inflector. So, the recommendation should be to use the new component instead.